pkg/cmd/build/view.go

```diff
@@ -41,6 +41,7 @@ func NewCmdBuildView(f *factory.Factory) *cobra.Command {
 			resolvers := resolver.NewAggregateResolver(
 				resolver.ResolveFromPositionalArgument(args, 1, f.Config),
 				resolver.ResolveFromPath("", f.Config.Organization, f.RestAPIClient),
+				resolver.ResolveFromConfig(f.LocalConfig),
```

Comment:
I think we want these above the ResolveFromPath

Reply:
Need to apply this to all the commands

Reply:
missed to save the other commands and didn't double check the commit 😓